config/ground-truth-yaml-files/all_features_version_09_no_visit_variant.yml

```diff
@@ -1996,7 +1996,7 @@ patient:
     maximum: 5
     minimum: 1
     type: integer
-  Days_DrugStartToEarliestSign_2:
+  Days_DrugStartToEarliestSign:_2:

```

Comment:
Is this the intended change? Looks like a typo to me @karafecho

Reply:
No, this is an older typo (the colon after "Sign" should be removed).

Reply:
Yeah, that is what I thought, but the change you added actually added the colon while the original file does not have the colon, right?

Reply:
Yeah, I'm thinking that the colon was removed after I downloaded the two version_09 files, so perhaps this is a timing thing. I should note that the version_09 files I am requesting to merge should be nearly identical to the existing files. If that is not the case, then please let me know.

Reply:
I think you can leave all_features_version_09_no_visit_variant.yml as is since the latest file in the master branch does not have colon for the variable which is what we want. I think if you remove the colon in this file in your local computer, then commit to github, we should be good. @karafecho

Reply:
Sounds good. Let's merge.

Reply:
@karafecho You will need to remove this file from your PR before we can merge since the updated file you committed has a colon in the variable which will cause us issues. You can remove the colon from this file and commit it again, which should fix this issue.

Reply:
I believe I removed the file from my PR, so we should be good to go.